x-pack/plugins/apm/public/components/app/mobile/transaction_overview/index.tsx

```diff
@@ -78,7 +78,7 @@ export function MobileTransactionOverview() {
         <TransactionsTable
           hideViewTransactionsLink
           numberOfTransactionsPerPage={25}
-          showAggregationAccurateCallout
+          showMaxTransactionGroupsExceededWarning
```

Comment:
Did we decide to not show the warning for mobile? Or what is the reasoning for not setting `showMaxTransactionGroupsExceededWarning` ?

Reply:
No, this setting is used for the `compacted representation` of the table we use in service overview
<img width="1321" alt="image" src="https://user-images.githubusercontent.com/1313018/215491592-1a2816c8-fccb-4186-81a4-12dbd01b7939.png">